client/src/pages/private/ParticipantTableDialogues.js

```diff
@@ -161,6 +161,7 @@ export const ParticipantTableDialogues = ({
             handleEngage(actionMenuParticipant.id, 'archived', values);
           }}
           onClose={onClose}
+          participantId={actionMenuParticipant?.id}
```

Comment:
Here you can send hole participant data. And `ros` info is included.

Reply:
When I pass in the participant here, this is what I get (for component props):
I didn't find any ROS information here nested in any of the properties.

{
  "onSubmit": "ƒ onSubmit() {}",
  "onClose": "ƒ handleDialogueClose() {}",
  "participantId": 27,
  "**participant**": {
    "nonHCAP": "yes",
    "crcClear": "yes",
    "lastName": "Everready",
    "location": "{coordinates: Array(2), type: \"Point\"}",
    "firstName": "Emma",
    "maximusId": 1000026,
    "interested": "yes",
    "postalCode": "V5X1A0",
    "phoneNumber": "6050987654",
    "emailAddress": "em1990@test.ca",
    "postalCodeFsa": "V5X",
    "userUpdatedAt": "2022-04-14T20:19:52.499Z",
    "callbackStatus": false,
    "preferredLocation": "Fraser",
    "id": 27,
    "created_at": "2022-04-14T20:19:55.474Z",
    "updated_at": null,
    "cohort": "{cohortParticipant: Array(1), cohort_name: \"Roomy C…}",
    "cohortName": "Roomy Cohort / Insitute Name",
    "postHireStatus": "{created_at: \"2022-05-11T17:34:13.530Z\", data: {…},…}",
    "postHireStatusLabel": "Graduation Completed on - 2022/02/02"
  }
}